Runtime/material/stepper.cs

```diff
@@ -0,0 +1,620 @@
+                        color: this._isDark() ? this._kErrorDark : this._kErrorLight
+                    );
+            }
+
+            return null;
+        }
+
+        TextStyle _subTitleStyle(int index) {
+            ThemeData themeData = Theme.of(this.context);
+            TextTheme textTheme = themeData.textTheme;
+
+            switch (this.widget.steps[index].state) {
+                case StepState.indexed:
+                case StepState.editing:
+                case StepState.complete:
+                    return textTheme.caption;
+                case StepState.disabled:
+                    return textTheme.caption.copyWith(
+                        color: this._isDark() ? this._kDisabledDark : this._kDisabledLight
+                    );
+                case StepState.error:
+                    return textTheme.caption.copyWith(
+                        color: this._isDark() ? this._kErrorDark : this._kErrorLight
+                    );
+            }
+
+            return null;
+        }
+
+        Widget _buildheaderText(int index) {
+            List<Widget> children = new List<Widget> {
+                new AnimatedDefaultTextStyle(
+                    style: this._titleStyle(index),
+                    duration: ThemeUtils.kThemeAnimationDuration,
+                    curve: Curves.fastOutSlowIn,
+                    child: this.widget.steps[index].title
+                )
+            };
+
+            if (this.widget.steps[index].subtitle != null) {
+                children.Add(
+                    new Container(
+                        margin: EdgeInsets.only(top: 2.0f),
+                        child: new AnimatedDefaultTextStyle(
+                            style: this._subTitleStyle(index),
+                            duration: ThemeUtils.kThemeAnimationDuration,
+                            curve: Curves.fastOutSlowIn,
+                            child: this.widget.steps[index].subtitle
+                        )
+                    )
+                );
+            }
+
+            return new Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisSize: MainAxisSize.min,
+                children: children
+            );
+        }
+
+        Widget _buildVerticalHeader(int index) {
+            return new Container(
+                margin: EdgeInsets.symmetric(horizontal: 24.0f),
+                child: new Row(
+                    children: new List<Widget> {
+                        new Column(
+                            children: new List<Widget> {
+                                this._buildLine(!this._isFirst(index)),
+                                this._buildIcon(index),
+                                this._buildLine(!this._isLast(index))
+                            }
+                        ),
+                        new Container(
+                            margin: EdgeInsets.only(12.0f), // EdgeInsetsDirectional
```

Comment:
This comments (i.e., //EdgeInsetsDirectional) maybe not necessary here. We will not add the TextDirection feature into UIWidgets in the future